docs/getting-started/apple/setup.md

```diff
@@ -1,7 +1,46 @@
-# Getting started with fastlane for iOS
+# Getting started with `fastlane` for iOS
 
-To be written soon, this will contain
+## Installing fastlane
 
-- Installing fastlane
-- Initial setup of fastlane for your projet
-- Where to go from there
+```no-highlight
+sudo gem install fastlane --verbose
+```
+
+## Setting up fastlane
+
+Navigate your terminal to your project's directory and run
+
+```no-highlight
+fastlane init
+```
+
+You'll be asked for your Apple ID, and `fastlane` will automatically generate a fastlane configuration for you, based on the information provided.
```

Comment:
We haven't made a decision yet, right? https://github.com/fastlane/docs/issues/36